model/HintList.ts

```diff
@@ -1,33 +1,53 @@
-
-
+/**
+ * This class represents a list of hints necessary for one game. It contains 6 hints, each of them being of type `Hint`.
+ * The hints are the birthdate, the death date, the occupation, the citizenship, the initials and the first paragraph
+ * of the Wikipedia page of the celebrity.
+ */
 export class HintList {
-    birth: Hint<Date> = new Hint(1,  true);
-    death : Hint<Date> = new Hint(1);
-    occupation : Hint<string>= new Hint(1);
-    citizenship : Hint<string> = new Hint(1);
-    initials : Hint<string> = new Hint(2);
-    paragraph1 : Hint<string> = new Hint (2);
+    birth: Hint<Date> ;
+    ageOrDeath : Hint<Date | number> ;
+    occupation : Hint<string> ;
+    citizenship : Hint<string> ;
+    initials : Hint<string> ;
+    paragraph : Hint<string> ;
+
+    constructor(
+        birth: Date | undefined ,
+        ageOrDeath: Date | number | undefined,
+        occupation: string | undefined,
+        citizenship: string | undefined,
+        initials: string | undefined,
+        paragraph: string | undefined
+    ){
+        this.birth = new Hint(1,  true, birth)
+        this.ageOrDeath = new Hint(1, true, ageOrDeath);
+        this.occupation = new Hint(1, true, occupation);
+        this.citizenship = new Hint(1, true, citizenship);
+        this.initials = new Hint(2, true, initials);
+        this.paragraph = new Hint(2, true, paragraph);
+    }
```

Comment:
revealed should be set to false first, apart from the birth which we show directly. 
Moreover I do not like "ageOrDeath" maybe it could be "aliveOrDeath" so that we tell the user if the celebrity is alive or we give the death date. Giving the age doesn't give any new hints since it is an information they already have from the birth date